internal/pkg/git/git_test.go

```diff
@@ -3,149 +3,149 @@ package git_test
 
 import (
 	"bytes"
-	"fmt"
-	"io/ioutil"
-	"os"
 	"os/exec"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/filesystem/aferofs"
-	"github.com/keboola/keboola-as-code/internal/pkg/git"
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	. "github.com/keboola/keboola-as-code/internal/pkg/git"
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
-	"github.com/keboola/keboola-as-code/internal/pkg/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/template/repository"
 )
 
 func TestGit_Available(t *testing.T) {
 	t.Parallel()
 
 	// should be always true as git is available in the container running the tests
-	assert.True(t, git.Available())
+	assert.True(t, Available())
 }
 
-func TestGit_CheckoutTemplateRepositoryPartial_Remote(t *testing.T) {
+func TestGit_Checkout(t *testing.T) {
 	t.Parallel()
+	logger := log.NewDebugLogger()
```

Comment:
A testy, ...zostalo to, co nebolo presunute do `cli/dependencies`, tj. iba core funkcionalita.